src/components/Mainvisual.jsx

```diff
@@ -0,0 +1,11 @@
+export const mainvisual = () => {
```

Comment:
export する component は、大文字で始めるべきです。

```suggestion
export const Mainvisual = () => {
```